projects/experimental/components/compass/compass.style.less

```diff
@@ -0,0 +1,32 @@
+@import 'taiga-ui-local';
+
+:host {
+    position: relative;
+    display: inline-block;
+    width: 1.1875rem;
+    height: 1.1875rem;
+    border-color: var(--tui-base-02);
+    color: var(--tui-primary);
+
+    &:before {
+        content: '';
+        position: absolute;
+        top: 0.21875rem;
+        left: 0.21875rem;
+        width: 0.75rem;
+        height: 0.75rem;
+        background: currentColor;
+        mask-image: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg"><path d="M0 8.99993C4 8 8 8 12 8.99993C9.91509 5.73239 8.5 3 6 0C3.5 3 2 5.5 0 8.99993Z"/></svg>');
+        transform: rotate(var(--t-degrees)) translateY(-0.625rem);
+    }
+
+    &:after {
+        .fullsize(absolute, inset);
+        content: '';
+        border-radius: 50%;
+        border: 0.1875rem solid;
+        border-color: inherit;
+        box-shadow: 0 0 0.4375rem rgba(0, 0, 0, 0.3);
```

Comment:
Sorry, `border-radius` was necessary on `:host` :) We should keep `box-shadow` there in case people need to remove it as per spec for dark backgrounds.

Reply:
fixed